bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/PowerControlPower.java

```diff
@@ -29,31 +33,81 @@
  */
 @NonNullByDefault
 public class PowerControlPower extends BaseChannelHandler<CommandConfirmation> {
+    private static final int WOL_PACKET_RETRY_COUNT = 10;
+    private static final int WOL_PACKET_RETRY_DELAY_MILLIS = 100;
+
     private final Logger logger = LoggerFactory.getLogger(PowerControlPower.class);
+    private final ConfigProvider configProvider;
+    private final ScheduledExecutorService scheduler;
+
+    public PowerControlPower(ConfigProvider configProvider, ScheduledExecutorService scheduler) {
+        this.configProvider = configProvider;
+        this.scheduler = scheduler;
+    }
 
     @Override
     public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command command) {
+        final State state = handler.getSocket().getState();
         if (RefreshType.REFRESH == command) {
             handler.postUpdate(channelId, handler.getSocket().isConnected() ? OnOffType.ON : OnOffType.OFF);
-            return;
-        }
-        if (OnOffType.ON == command) {
-            logger.debug(
-                    "Received ON - Turning TV on via API is not supported by LG WebOS TVs. You may succeed using wake on lan binding, please consult lgwebos binding documentation.");
-        }
-        if (!handler.getSocket().isConnected()) {
-            /*
-             * Unable to send anything to a not connected device.
-             * onDeviceReady nor onDeviceRemoved will be called and item state would be permanently inconsistent.
-             * Therefore setting state to OFF
-             */
-            handler.postUpdate(channelId, OnOffType.OFF);
+        } else if (OnOffType.ON == command) {
+            if (State.DISCONNECTED == state) {
+                String macAddress = configProvider.getMacAddress();
+                if (macAddress.isEmpty()) {
+                    logger.debug(
+                            "Received ON - Turning TV on via API is not supported by LG WebOS TVs. You may succeed using wake on lan (WOL). Please set the macAddress config value in Thing configuration to enable this.");
```

Comment:
can you run the spotless formatter on this? This string is too long and should be split into multiple lines.

Reply:
can you give a hint how to do this? spotless formatter is completely new to me

Reply:
This is how you would do it, but it would apply to the whole project not just that file.
`mvn spotless:apply -p1 :org.openhab.binding.lgwebos`
You don't need to commit all the changes it makes, I'm more just concerned about the formatting of the `PowerControlPower` file. Feel free to ignore changes it makes to other files since I feel it would be outside the scope of this PR.

Reply:
this has only changed xml files
```
	modified:   pom.xml
	modified:   src/main/feature/feature.xml
	modified:   src/main/resources/ESH-INF/config/config.xml
```

?

Reply:
Well, I guess I misunderstood what it would change... nvm then. I still need the string to less long so just do this:
```suggestion
                            "Received ON - Turning TV on via API is not supported by LG WebOS TVs. " 
                            + "You may succeed using wake on lan (WOL). "
                            +"Please set the macAddress config value in Thing configuration to enable this.");
```